pydruid/query.py

```diff
@@ -159,6 +153,12 @@ def export_pandas(self):
                     0      7  2013-10-04T00:00:00.000Z         user_1
                     1      6  2013-10-04T00:00:00.000Z         user_2
         """
+        try:
+            import pandas
+        except ImportError:
+            print('Warning: unable to import Pandas. The export_pandas method will not work.')
```

Comment:
we should change this to be an error instead of a warning in this case, since we are raising an exception